BankManager.swift

```diff
@@ -1,7 +1,65 @@
 //
 //  BankManager.swift
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
+//  BankManagerConsoleApp
+//
+//  Created by OneTool, Red on 2022/04/28.
 //
 
 import Foundation
+
+fileprivate extension Const {
+    static let bankersNumber = 3
+    
+    static let blank = ""
+    static let OpeningInput = "1"
+    static let exitInput = "2"
+    
+    static let startBankSelect = "1 : 은행 개점\n2 : 종료\n입력 : "
+    static let wrongInput = "잘못된 입력 입니다."
+    static let finishWork = "업무가 마감 되었습니다."
+    
+}
+
+fileprivate enum UserChoice: Int {
+    case start = 1
+    case exit = 2
+}
+
+protocol Manageable {
+    func manageBanker()
+    func reportOfDay() -> String
+}
+
+final class BankManager {
+    var bank: Manageable = Bank(numberOfBankers: Const.bankersNumber)
+    
+    func openBank() {
+        var result = true
+        while result == true {
```

Comment:
```suggestion
     whilte workBank() { }
```

위와 같이 변경하면 불필요한 변수는 안생길거같아요

Reply:
넵! 위와 같이 수정했습니다!